corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -405,8 +405,17 @@ hqDefine("cloudcare/js/formplayer/app", function () {
     FormplayerFrontend.on('navigation:back', function () {
         var url = Backbone.history.getFragment();
         if (!url.includes('single_app')) {
-            window.history.back();
+            return;
+        }
+        try {
+            var options = JSON.parse(url);
+            if (_.has(options, "endpointId")) {
+                return;
+            }
+        } catch (e) {
+            // do nothing
         }
+        window.history.back();
```

Comment:
this seems to invert the existing functionality (go back unless url includes 'single_app')

Reply:
🤦  thanks

e3a4938e379